src/shared/post/Post.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import Avatar from "../../assets/images/avatar.jpg";
+import MoreOptions from "../../shared/moreOptions/MoreOptions";
+import {
+  BsThreeDots,
+  BsHeartFill,
+  MdInsertComment,
+  BsBookmarkFill,
+} from "../../assets/icons/icons";
+import "./post.css";
+
+const Post = () => {
+  const [show, setShow] = useState(false);
+
+  const handleMoreOptions = () => {
+    setShow(!show);
+  };
+  return (
+    <div className="flex-col post-container">
+      <div className="flex-row">
+        <img src={Avatar} className="avatar small-avatar" alt="profile" />
+        <div className="flex-col post-header ml-1">
+          <div className="flex-row align-center justify-between">
+            <h3 className="username">Priya Patel</h3>
+            <div className="more-options-wrapper">
+              <BsThreeDots
+                className="more-icon"
+                onClick={() => handleMoreOptions()}
```

Comment:
fix: you can just write handleMoreOptions() instead of () => handleMoreOptions()